k-distribution/tests/regression-new/group/test.k

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) K Team. All Rights Reserved.
+module TEST-SYNTAX
+  imports INT-SYNTAX
+
+  syntax Int ::=
+      "(" Int ")" [bracket]
+    | "-" Int     [function, unmin]
```

Comment:
This test just confirms that the group(_) attributes are appropriately handled and can actually be referenced by e.g. syntax priorities (the other tests error out so they don't confirm this).

Good point that this will fail with `--pedantic-attributes` though, which will eventually be required, so I'll clean it up